Gruntfile.js

```diff
@@ -1,9 +1,9 @@
 "use strict";
 
 var async = require( "async" ),
-	fs = require( "fs" ),
-	path = require( "path" ),
-	rimraf = require( "rimraf" ),
+	fs = require( "node:fs" ),
+	fsp = require( "node:fs/promises" ),
```

Comment:
Could all calls be switched to use one or the other?

Reply:
There are sync-only APIs like `existsSync` or `createWriteStream` so `node:fs/promises` cannot be used for everything. There are some cases that could potentially be migrated but validating I didn't break anything would require some time. I'd rather postpone any code refactors until after jQuery UI <1.12 support is dropped from this repo.

Reply:
@timmywil I’m going to merge this since I have 1 approval and this blocks multiple other PRs. Let me know if you have further feedback and I’ll address it separately.